fv3core/stencils/saturation_adjustment.py

```diff
@@ -113,49 +96,41 @@ def qs_table_fn(i):
 @gtscript.function
 def qs_table2_fn(i):
     tem0 = tem_lower(i)
-    table2 = 0.0
     if i < 1600:
         # compute es over ice between - 160 deg c and 0 deg c.
         table2 = table_ice_oneline(tem0)
     else:
         # compute es over water between 0 deg c and 102 deg c.
         table2 = table_vapor_oneline(tem0)
-    table2_m1 = 0.0
-    table2_p1 = 0.0
     # TODO is there way to express the code below with something closer to this?:
     # if i == 1599:
     #    table2 = 0.25 * (table2(i-1) + 2.0 * qs_table_fn(i) + qs_table2_fn(i+1))
     # if i == 1600:
     #    table2 = 0.25 * (table2(i-1) + 2.0 * qs_table_fn(i) + qs_table2_fn(i+1))
```

Comment:
Do these work? (might not be correct). Probably not, though what's below looks closer. We can probably delete this comment

Reply:
I tried these and they didn't work. The assignment to `table2` makes a recursive call to `qs_table2_fn`, and my thought was that GT4Py wasn't capable of executing recursive code since the error says `Unknown symbol "qs_table2_fn"`